src/org/usfirst/frc/team694/robot/commands/auton/SameSideRightAngleAutonCommand.java

```diff
@@ -0,0 +1,39 @@
+package org.usfirst.frc.team694.robot.commands.auton;
+
+import org.usfirst.frc.team694.robot.FieldMap;
+import org.usfirst.frc.team694.robot.commands.DrivetrainRotateDegreesPIDCommand;
+import org.usfirst.frc.team694.robot.commands.GrabberOpenCommand;
+import org.usfirst.frc.team694.robot.commands.LiftMaxUpCommand;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ *
+ */
+public class SameSideRightAngleAutonCommand extends CommandGroup {
```

Comment:
This branch should not be including these Auton Commands, make sure you pull before you push.